src/utils/index.ts

```diff
@@ -59,6 +59,10 @@ function getEtherscanDomain(chainId: ChainId): string {
       return 'https://polygonscan.com'
     case ChainId.MUMBAI:
       return 'https://explorer-mumbai.maticvigil.com'
```

Comment:
please help update it to `https://mumbai.polygonscan.com`

Reply:
agree